ui/app/styles/core/link.scss

```diff
@@ -21,7 +21,7 @@
     cursor: default;
   }
 }
-// ARG TODO: nice-to-do replace all instances with helper is-no-underline
+// ARG TODO: nice-to-do replace all instances with helper
```

Comment:
I did do this and then reverted the change. I'm going to avoid any hbs changes I can in this sidebranch. If we determine that some of these nice to haves are worth it I can merge into main later, avoiding large conflicts while trying to maintain a sidebranch.